bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/HomeConnectBindingConstants.java

```diff
@@ -0,0 +1,214 @@
+    public static final String STATE_AMBIENT_LIGHT_COLOR_CUSTOM_COLOR = "BSH.Common.EnumType.AmbientLightColor.CustomColor";
+
+    // List of programs
+    public static final String PROGRAM_HOOD_AUTOMATIC = "Cooking.Common.Program.Hood.Automatic";
+    public static final String PROGRAM_HOOD_VENTING = "Cooking.Common.Program.Hood.Venting";
+    public static final String PROGRAM_HOOD_DELAYED_SHUT_OFF = "Cooking.Common.Program.Hood.DelayedShutOff";
+
+    // Network and oAuth constants
+    public static final String API_BASE_URL = "https://api.home-connect.com";
+    public static final String API_SIMULATOR_BASE_URL = "https://simulator.home-connect.com";
+    public static final String OAUTH_TOKEN_PATH = "/security/oauth/token";
+    public static final String OAUTH_AUTHORIZE_PATH = "/security/oauth/authorize";
+    public static final String OAUTH_SCOPE = "IdentifyAppliance Monitor Settings Dishwasher-Control Washer-Control Dryer-Control WasherDryer-Control CoffeeMaker-Control Hood-Control CleaningRobot-Control";
+
+    // Operation states
+    public static final String OPERATION_STATE_INACTIVE = "BSH.Common.EnumType.OperationState.Inactive";
+    public static final String OPERATION_STATE_READY = "BSH.Common.EnumType.OperationState.Ready";
+    public static final String OPERATION_STATE_DELAYED_START = "BSH.Common.EnumType.OperationState.DelayedStart";
+    public static final String OPERATION_STATE_RUN = "BSH.Common.EnumType.OperationState.Run";
+    public static final String OPERATION_STATE_PAUSE = "BSH.Common.EnumType.OperationState.Pause";
+    public static final String OPERATION_STATE_ACTION_REQUIRED = "BSH.Common.EnumType.OperationState.ActionRequired";
+    public static final String OPERATION_STATE_FINISHED = "BSH.Common.EnumType.OperationState.Finished";
+    public static final String OPERATION_STATE_ERROR = "BSH.Common.EnumType.OperationState.Error";
+    public static final String OPERATION_STATE_ABORTING = "BSH.Common.EnumType.OperationState.Aborting";
+
+    // Commands
+    public static final String COMMAND_START = "start";
+    public static final String COMMAND_STOP = "stop";
+    public static final String COMMAND_SELECTED = "selected";
+    public static final String COMMAND_VENTING_1 = "venting1";
+    public static final String COMMAND_VENTING_2 = "venting2";
+    public static final String COMMAND_VENTING_3 = "venting3";
+    public static final String COMMAND_VENTING_4 = "venting4";
+    public static final String COMMAND_VENTING_5 = "venting5";
+    public static final String COMMAND_VENTING_INTENSIVE_1 = "ventingIntensive1";
+    public static final String COMMAND_VENTING_INTENSIVE_2 = "ventingIntensive2";
+    public static final String COMMAND_AUTOMATIC = "automatic";
+    public static final String COMMAND_DELAYED_SHUT_OFF = "delayed";
+
+    // proxy settings
+    public static final boolean HTTP_PROXY_ENABLED = false;
```

Comment:
Doesn't seem to do much.
It would make more sense to get the proxy configuration from where the rest of OpenHab also get its from, so all components behave uniformly.

Reply:
Removed it